lib/extensions/acts_as_helpers/active_record/base.rb

```diff
@@ -28,5 +28,18 @@ def experience_points_display_reason
       raise NotImplementedError,
             'Experience points awarding items need to implement a points_display_reason method'
     end
+
+    # The parameters which will be resolved to the URL that the displayed reason links to on
+    # the experience points history page.
+    # This method is meant to be overridden by each class that acts as an experience points record.
+    #
+    # @return [Array|Object] The resource to link to
+    # @return [String] The path to link to
+    # @return [nil] If there is no link
```

Comment:
Not sure if @allenwq prefers everything on a single `return`, e.g. `@return [Array|Object|String|nil]`.

See his comment on the outdated diff for https://github.com/Coursemology/coursemology2/pull/1469